var/spack/repos/builtin/packages/gdbm/package.py

```diff
@@ -27,11 +28,11 @@ class Gdbm(AutotoolsPackage, GNUMirrorPackage):
     version('1.9',    sha256='f85324d7de3777db167581fd5d3493d2daa3e85e195a8ae9afc05b34551b6e57')
 
     depends_on("readline")
-    patch('gdbm.patch', when='@:1.18 %gcc@10:')
-    patch('gdbm.patch', when='@:1.18 %clang@11:')
-    patch('gdbm.patch', when='@:1.18 %cce@11:')
-    patch('gdbm.patch', when='@:1.18 %aocc@2:')
-    patch('gdbm.patch', when='@:1.18 %oneapi')
+    patch('gdbm.patch', when='@:1.18.1%gcc@10:')
+    patch('gdbm.patch', when='@:1.18.1%clang@11:')
+    patch('gdbm.patch', when='@:1.18.1%cce@11:')
+    patch('gdbm.patch', when='@:1.18.1%aocc@2:')
+    patch('gdbm.patch', when='@:1.18.1%oneapi')
```

Comment:
this is not necessary since :1.18 is inclusive -- covers everything < 1.19. but also it doesn't hurt.

Reply:
I didn't know that. If possible, I would prefer not to revert it.